src/Monolog/Handler/HipChatHandler.php

```diff
@@ -93,6 +106,22 @@ public function __construct($token, $room, $name = 'Monolog', $notify = false, $
         $this->room = $room;
         $this->format = $format;
         $this->host = $host;
+
+        $this->setVersion($version);
+    }
+
+    /**
+     * Convenience method for setting the API version. Valid values are
+     * - HipChatHandler::API_V1
+     * - HipChatHandler::API_V2
+     * @param $version string the API version to use
+     */
+    public function setVersion($version) {
```

Comment:
I would assume it's so the user doesn't have to specify all of the default arguments just to use v2?